pint/models/glitch.py

```diff
@@ -8,13 +8,13 @@
     from astropy.erfa import DAYSEC as SECS_PER_DAY
 except ImportError:
     from astropy._erfa import DAYSEC as SECS_PER_DAY
-from .parameter import Parameter, MJDParameter
+from .parameter import Parameter, MJDParameter,prefixParameter
```

Comment:
I often see a lot of python style issues in your code (like the missing space after the comma here, or typos in documentation / comments.  I highly recommend that you run your code through "pep8" before you push it.  In fact, it would be great if you could do that now for this pull request!